controllers/ie_controller.go

```diff
@@ -11,10 +11,10 @@ import (
 
 // IEController Wrapped for Patient Collection
```

Comment:
There are still patient-specific comments in what I think is intended to be a _generic_ controller.